CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contribution Guide #
+# Contribution Guide
```

Comment:
nice catch. There is nothing offending me. Actually, this is my problem which is I did not did thorough research. Thanks.